### DISCUSSION TITLE
How do I add a subdomain to ignored site list?

### DISCUSSION THREAD

**Discussion by carloswm85** [darkreader/darkreader#6700] [General]
Here's the version I have: 
![image](https://user-images.githubusercontent.com/29048235/132142360-1445591e-79e5-44df-bca8-06191a53fb03.png)
At Google Chrome, Version 93.0.4577.63 (Official Build) (64-bit)

I have DR enabled for sites at `https://www.google.com/`, which is what I want. But I'd like to DISable it at the subdomain `https://www.google.com/maps/`. I've tried using some of the settings at the extension, but unsuccessfully. Any help with this?

For example, I've tried going to `DR → Settings → Site List`, but the solution seems to be not there.

![image](https://user-images.githubusercontent.com/29048235/132142430-106b4cd1-1403-4828-a5b1-170a53793b3a.png)

**Comment by Gusted** [comment]
What about adding `www.google.com$`?

**Reply by carloswm85** [reply]
> What about adding `www.google.com$`?

Not working. Sadly.

What is that suppose `$` to do?

Is there something like counter-rules I can add? I want it to work only in pages like `https://www.google.com/maps/@-38.9080543,-70.0855921,15z` That town for example gets (almost) completely dark.

![image](https://user-images.githubusercontent.com/29048235/132270508-144ebd56-2580-4109-ad56-89afb946585a.png)

I want it to not do that at `https://www.google.com/maps/` but I want dark mode working on any other Google subdomain.

**Reply by Gusted** [reply]
> What is that suppose `$` to do?

It should only match URL's, that ends in that URL.

> Is there something like counter-rules I can add? I want it to work only in pages like `https://www.google.com/maps/@-38.9080543,-70.0855921,15z` That town for example gets (almost) completely dark.

No currently not, see #2517
